src/components/JobsTrainingButtons/index.jsx

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import { useTranslation } from "react-i18next";
+import "./index.scss";
+
+const JobsTrainingButtons = ({ setCardsArr, setSearchCards }) => {
+  const { t } = useTranslation();
+  //The selected button is disabled to let the user know which data he is viewing (jobs or training).
+  const [isDisabled, setIsDisabled] = useState(true);
+
+  // If a button is disabled it means it's data is already fetched. So, on the click of the other button this function detects the disabled button and fetches the opposite data.
+  const fetchOpportunityData = () => {
+    if (isDisabled) {
+      setSearchCards([...t(`home.trainings`, { returnObjects: true })]);
+      setCardsArr([...t(`home.trainings`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    } else {
+      setSearchCards([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setCardsArr([...t(`home.jobOpportunities`, { returnObjects: true })]);
+      setIsDisabled(!isDisabled);
+    }
+  };
+
+  const buttonNames = ["Job Opportunities", "Trainings / Workshops"];
+  // This variables maps the two button Names and returns a button for each. One for Jobs and one for training to fetch data accordingly.
+  const opportunityListBtns = buttonNames.map((name, index) => {
+    return (
+      <button
+        key={name}
+        type="button"
+        className="opportunityListBtn"
+        onClick={() => {
+          fetchOpportunityData();
+        }}
+        disabled={index === 0 ? isDisabled : !isDisabled}
```

Comment:
so if it's the first button its disabled? again i think your state is not named what you want it to be. take a read at this code:
```
disabled={!isDisabled}
```
(the second part of your condition)

this button is disabled when it's not disabled?

Reply:
It's like this :
this first button is disabled when the second is not disabled, does it make sense? 

If jobs button (disabled={isDisabled} / disabled={true} ) then training button is (disabled={!isDisabled} / disabled={false}). and vice versa.

Reply:
what I wrote is about the two buttons but this one line disabled={false} maybe I can make it false and true in this line ?

Reply:
I am talking about how the two buttons work.

Reply:
maybe I should change the naming to clicked notClicked ?

Reply:
So i did the disabled according to index, but i could't manage to do the fetch according to the index :/ . so I added another state for  isTrainingSelected or isJobsSelected. But I am not sure it's the correct way ?